### PR TITLE
Support for include.gradle from plugins

### PR DESCRIPTION
**Pull request by Fatme**
https://github.com/NativeScript/nativescript-cli/issues/1048

This PR should be merged after the corresponding in android runtime.

### CODE REVIEW

File: lib/project-data.ts [hunk 1]
```diff
@@ -101,6 +103,8 @@ export class ProjectData implements IProjectData {
 		this.projectName = this.$projectHelper.sanitizeName(path.basename(projectDir));
 		this.platformsDir = path.join(projectDir, "platforms");
 		this.projectFilePath = path.join(projectDir, this.$staticConfig.PROJECT_FILE_NAME);
+		this.appDirectoryPath = path.join(projectDir, "app");
+		this.appResourcesDirectoryPath = path.join(projectDir, "app", "App_Resources");
```
Comment (ligaz) [inline lib/project-data.ts]: There is a constant for `App_Resources`.

File: lib/project-data.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ///<reference path=".d.ts"/>
 "use strict";
 
+import constants = require("./constants");
```
Comment (teobugslayer) [inline lib/project-data.ts]: Please use ES6 imports

**Comment by teobugslayer** [comment]
:+1: squash and go

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**